curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/create-strings-using-template-literals.english.md

```diff
@@ -32,8 +33,9 @@ tests:
   - text: <code>resultDisplayArray</code> is the desired output.
     testString: assert(makeList(result.failure).every((v, i) => v === `<li class="text-warning">${result.failure[i]}</li>` || v === `<li class='text-warning'>${result.failure[i]}</li>`), '<code>resultDisplayArray</code> is the desired output.');
   - text: Template strings were used
```

Comment:
IMO, we should rephrase this as well. For example: `You need to use a template string` or `Template string should be used`. And use it in the `testString`.

We should describe what result we want to get.

Reply:
How does this sound? "Template string should be used" in the `text` and in `textString` - "Template string or interpolation wasn't used". Could use suggestion here. 😅

Reply:
In the most challenges `text` and `textString` contain the same string.

Reply:
So, "Template string should be used" is fine?

Reply:
yes

Reply:
and for the next test also.

Reply:
Fixed.

Reply:
> in most challenges `text` and `testString` are the same

I noticed that, it's wrong/lazy isn't it? `text` should be user storyish requirement and `testString` is only displayed on failure, so it should explain what is wrong? 
Impractical to fix them all, but we could fix the ones we update?

Reply:
I guess its for the convenience but yes I agree it should be a bit more detailed explaining what is wrong with the code.

Reply:
If you see an error message: `Template string should be used`, what information is missing to understand what is wrong?

Reply:
IMHO. A user story should describe what we want from campers and a test should check it accordingly. If you can't understand why you don't pass a test then the story is bad or the test is wrong.

Reply:
Ok - so we dont want solution like this ``arr.map(item => `<li class="text-warning">`+item+`</li>`);`` to the pass the tests but it does technically use "Template string". Shouldnt we mention that "string interpolation" should also be used?

Reply:
Done.